src/app/client/src/app/modules/workspace/components/content-editors/collection-editor/collection-editor.component.ts

```diff
@@ -40,6 +40,7 @@ export class CollectionEditorComponent implements OnInit, OnDestroy {
   public ownershipType: Array<string>;
   public queryParams: object;
   resource_framework: string;
+  collectionEditorCDN: string = (<HTMLInputElement>document.getElementById('collectionEditorCDN')).value;
```

Comment:
Create the service to take any HTMLInputElement values. Especially for Env values of indec.html.
Components will call only get methods of the service. Dont pass the variable name also from the component.

example:
service: {
getContentEditorPath () {
 return getValue('collectionEditorCDN');
}
getValue(propertyName) {
return (<HTMLInputElement>document.getElementById(propertyName')).value;
}
}

Reply:
Dont depclare the variable on component level. Define the variable in the initEditor function as use.